pandas/tests/indexing/test_indexing.py

```diff
@@ -5489,6 +5489,269 @@ def test_none_coercion_mixed_dtypes(self):
                                         strict_nan=True)
 
 
+class TestIndexingCallable(tm.TestCase):
+
+    _multiprocess_can_split_ = True
+
+    def test_frame_loc_ix_callable(self):
+        # GH 11485
+        df = pd.DataFrame({'A': [1, 2, 3, 4], 'B': list('aabb'),
+                           'C': [1, 2, 3, 4]})
+        # iloc cannot use bool mask
```

Comment:
hmm, IIRC there is an open issue for this. We should allow this (as we do for DataFrames). If not, can you create an issue? thxs